src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@import 'variables/variabrles';
+@import 'blocks/card';
+@import 'blocks/stars';
```

Comment:
leave only the imports in this file. 
move all other code to the `reset.scss` file